tests/postgres/vm.test.ts

```diff
@@ -14,6 +14,20 @@ jest.mock('../../src/utils/celery/client')
 jest.mock('../../src/main/job-queues/job-queue-manager')
 jest.setTimeout(30000)
 
+// passOnAnything is needed to match any object + undefined, null, etc.
+// expect.anything() will not match null and undefined
+const passOnAnything = () => ({
+    message: () => 'Ok',
+    pass: true,
+})
+
+const extendedExpect = expect as ExtendedExpect
+extendedExpect.extend({
+    passOnAnything,
+})
```

Comment:
Why is any of this needed? Can't we just use `expect.any()` or then explicitly check for `null` or `undefined` when needed?

Reply:
I actually had a problem where the tests were all testing `fetch` calls, and with the manual redirects testing, the test call would have with an options object `{ redirect: 'manual' }` and the actual call would have `undefined`. `expect.any()` nor `expect.anything()` were suitable. 

Another approach is to do `init || {}` to the actual fetch call to prevent it from being undefined. Then I guess `expect.any` could be used. 

Some discussion on solutions for matching `undefined` and "any object" https://stackoverflow.com/questions/47027011/using-expect-anyobject-or-expect-anything-does-not-work-to-match-undefi